app/views/account/projects/index.html.slim

```diff
@@ -1,7 +1,8 @@
 .container
   .level
     .level-left
-      .title My projects
+      .title 
+        = title "#{@workspace.name} projects:"
```

Comment:
title could be different from this

use title for <title> as a separate call at the top of the view file